src/CreatePackage.php

```diff
@@ -0,0 +1,238 @@
+
+        if ($answer !== 'c') {
+            return $namespaces[$answer];
+        }
+
+        $question = new Question('Custom namespace: ');
+        $question->setValidator(function ($value) use ($repo) {
+            if (strpos($repo, 'zf-') === 0 || strpos($repo, 'zend-') === 0) {
+                if (! preg_match('/^(ZF|Zend)(\\\\[A-Z][a-zA-Z0-9]*)+$/', $value)) {
+                    throw new RuntimeException(sprintf(
+                        'Invalid namespace provided: %s',
+                        $value
+                    ));
+                }
+            }
+
+            // must be consistent with package name
+            if (strtolower(str_replace('\\', '', $value))
+                !== strtolower(str_replace('-', '', $repo))
+            ) {
+                throw new RuntimeException('Namespace have to be consistent with package name');
+            }
+
+            return $value;
+        });
+
+        return $helper->ask($input, $output, $question);
+    }
+
+    private function copyFiles($src, $dest, array $replacement)
+    {
+        $dir = new RecursiveDirectoryIterator($src);
+        $iterator = new RecursiveIteratorIterator($dir);
+
+        $replacementJson = $replacement;
+        $replacementJson['{namespace}'] = str_replace('\\', '\\\\', $replacement['{namespace}']);
+        $replacementJson['{namespace-test}'] = str_replace('\\', '\\\\', $replacement['{namespace-test}']);
+
+        /** @var \RecursiveDirectoryIterator $iterator */
+        $iterator->rewind();
+        while ($iterator->valid()) {
+            if (! $iterator->isDot()) {
+                $file = $iterator->getSubPathname();
+                $dirname = dirname($dest . $file);
+                if (! is_dir($dirname)) {
+                    mkdir($dirname, 0775, true);
+                }
+
+                if ($iterator->isLink()) {
+                    symlink(readlink($src . $file), $dest . $file);
+                } else {
+                    $content = file_get_contents($src . $file);
+                    $content = strtr($content, $iterator->getExtension() === 'json' ? $replacementJson : $replacement);
+                    file_put_contents($dest . $file, $content);
+                }
+            }
+
+            $iterator->next();
+        }
+    }
+
+    private function getOrg($repo)
+    {
+        if (false !== strpos($repo, '/')) {
+            return explode('/', $repo, 2);
+        }
+
+        if (! preg_match('/^(?P<type>zend|zf)-/', $repo, $matches)) {
+            throw new RuntimeException('Missing organization in package name');
+        }
+
+        switch ($matches['type']) {
+            case 'zend':
+                return ['zendframework', $repo];
+            case 'zf':
+                return ['zfcampus', $repo];
+        }
+    }
+
+    private function getCategory($repo)
+    {
+        if (strpos('zf-', $repo)) {
+            return 'apigility';
+        }
+
+        if (strpos('zend-expressive', $repo)) {
```

Comment:
Maybe change this to `strpos('-expressive-', $repo)`; that way if somebody is building a package with expressive-related functionality, but in a different top-level namespace, it will still use this category.